### PR TITLE
update admin-ui to 1.5.1

### PR DESCRIPTION
**Pull request by m-kharbat**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -27,6 +27,11 @@ Breaking Changes
 Changes
 =======
 
+ - Added Privileges plugin for the Enterprise Edition.
```
Comment (mxm) [inline CHANGES.txt]: This is about adding a privileges tab to the admin ui, right? We could mention that in the change entry.

File: CHANGES.txt [hunk 2]
```diff
@@ -27,6 +27,11 @@ Breaking Changes
...
+
+ - The documentation link now points to the documentation for the major.minor
+   (e.g. 2.1) version of CrateDB, instead of major.minor.hotfix (e.g. 2.1.1).
```
Comment (mxm) [inline CHANGES.txt]: The "documentation link *in the admin ui*"

File: CHANGES.txt (hunk 1 above)
Comment (seut) [inline CHANGES.txt]: pls note that this is admin-ui related, also a user don't know about a 'plugin', I think its a main menu entry/section.

File: CHANGES.txt [hunk 3]
```diff
@@ -63,6 +68,9 @@ Changes
 Fixes
 =====
 
+ - Fixed issue that caused the Monitoring tab to redirect to ``/401`` when
```
Comment (seut) [inline CHANGES.txt]: pls also mention here  that this is admin-ui related.

**Pull request merged**